README.md

```diff
@@ -3,18 +3,35 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T1055324)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-# FileManager for Web Forms - How to open PDF and Images in popup
-This example demonstrates how to open images and PDF-files using ASPxFileManager, ASPxPanel and ASPxImage controls.
+
+# The file manager for Web Forms - How to open PDF files and images in a popup window
+
+This example demonstrates how to use the [ASPxFileManager](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxFileManager), [ASPxPanel](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPanel?p=netframework), and [ASPxImage](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxImage) controls to open **PDF** files and images in a [popup window](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl?p=netframework).
+
 ![Sample](./Sample.png)
- ## Overview
-The [ASPxFileManager.SelectedFileOpened](https://docs.devexpress.com/AspNet/js-ASPxClientFileManager.SelectedFileOpened) event is handled to open the file using ASPxImage to show images or ASPxPanel with [Embed tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/embed) to show PDF inside a popup.
+
 ## Files to Look At
-[Default.aspx](./CS/OpenPdfOrImage/Default.aspx)
+
+* [Default.aspx](./CS/OpenPdfOrImage/Default.aspx) (VB: [Default.aspx](./VB/OpenPdfOrImage/Default.aspx))
+
+* [Default.aspx.cs](./CS/OpenPdfOrImage/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/OpenPdfOrImage/Default.aspx.vb))
+
+## Implementation Details
+
+The [ASPxFileManager](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxFileManager) control allows users to manage files. This example handles the [ASPxFileManager.SelectedFileOpened](https://docs.devexpress.com/AspNet/js-ASPxClientFileManager.SelectedFileOpened) event to determine the opened file's type. 
+
+Depending on the file type, the handler manipulates the following controls contained in the [ASPxPopupControl](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl?p=netframework) control:
+
+1. The [ASPxImage](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxImage) control - Displays images.
```

Comment:
I don't like the punctuation here: the hyphen followed by a complete sentence, starting with a capital letter. 

You also better use an unordered list. The order or number of items are not important here.

```suggestion
- The [ASPxImage](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxImage) control. Displays images.
```

Reply:
Sounds good, thanks!